src/NuGet.Core/Microsoft.Build.NuGetSdkResolver/Strings.resx

```diff
@@ -120,9 +120,15 @@
   <data name="CouldNotFindInstalledPackage" xml:space="preserve">
     <value>Failed to resolve SDK '{0}'. A package was successfully restored but the package could not be located.</value>
   </data>
+  <data name="FailedToFindPathToGlobalJson" xml:space="preserve">
+    <value>Failed to determine path to global.json from path "{0}". {1}</value>
+  </data>
```

Comment:
There need to be comments telling translators what `{0}` and {1}` mean

Reply:
Thanks, I'll add comments for all of these!